DataConnectors/UniFiSG/Unifi_USG.json

```diff
@@ -0,0 +1,205 @@
+{
+    "id": "UnfiSG",
+    "title": "Unfi Security Gateway (Preview) ",
+    "publisher": "Azure Sentinel Community",
+    "descriptionMarkdown": "The UniFi Security Gateway extends the UniFi Enterprise System to provide cost-effective, reliable routing and advanced security for your network.  Unifi Security Gateway does not output CEF by default, Logstash is used to parse message into CEF for this connector. By connecting your CEF logs to Azure Sentinel, you can take advantage of search & correlation, alerting, and threat intelligence enrichment for each log.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "UnifiSG",
+            "baseQuery": "CommonSecurityLog\n| where DeviceVendor == \"Unifi\" and where DeviceProduct == \"USG\"\n"
+        },
+        {
+            "metricName":"Total data received",
+            "legend": "UnifiSG-Netflow",
+            "baseQuery": "netflow_CL\n"

```

Comment:
Please rename this datatype as unifinetflow_CL. Ultimately we'll try to bring in generic netflow logs as well so this might  conflict. Thanks.

Reply:
this is generic netflow.  unifi uses the industry standard netflow formats.